content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -419,13 +593,21 @@ An encoded Protobuf message with the following IDL:
   }
 ```
 
-Clients that receive a response in `application/vnd.kubernetes.protobuf` that does not match the expected prefix should reject the response, as future versions may need to alter the serialization format in an incompatible way and will do so by changing the prefix.
+{{< note >}}
+Clients that receive a response in `application/vnd.kubernetes.protobuf` that does
+not match the expected prefix should reject the response, as future versions may need
+to alter the serialization format in an incompatible way and will do so by changing
+the prefix.
+{{< /note >}}
 
 ## Resource deletion
 
-Resources are deleted in two phases: 1) finalization, and 2) removal.
+When you **delete** a resource this takes place in two phases.
 
-```go
+1. _finalization_
```

Comment:
Are the phases defined in this page?

Reply:
Can we save that for a follow-up? I don't think the original text describes these either.

Reply:
👍

Reply:
👍